aiohttp/cookiejar.py

```diff
@@ -82,8 +82,16 @@ def __iter__(self) -> 'Iterator[Morsel[str]]':
     def __len__(self) -> int:
         return sum(1 for i in self)
 
+    def _next_whole_second(self) -> datetime.datetime:
```

Comment:
This helper function should just be that: a function at module level.

Reply:
Moved.